docs/data/date-pickers/digital-clock/digital-clock.md

```diff
@@ -0,0 +1,80 @@
+---
+product: date-pickers
+title: React Digital Clock component
+components: DigitalClock, MultiSectionDigitalClock
+githubLabel: 'component: TimePicker'
+packageName: '@mui/x-date-pickers'
+---
+
+# Digital Clock
+
+<p class="description">The Digital Clock components let the user select a time without any input or popper / modal.</p>
+
+## Description
+
+There are two component versions for different cases. The `DigitalClock` handles selection of a single time instance in one step, just like a `select` component. The `MultiSectionDigitalClock` allows selecting time using separate sections for separate views.
+
+The `DigitalClock` is more appropriate when there is a limited amount of time options needed, while the `MultiSectionDigitalClock` is suited for cases when a more granular time selection is needed.
+
+## Basic usage
+
+{{"demo": "DigitalClockBasic.js"}}
+
+## Uncontrolled vs. Controlled
+
+The components can be uncontrolled or controlled.
+
+{{"demo": "DigitalClockValue.js"}}
+
+## Form props
+
+The components can be disabled or read-only.
+
+{{"demo": "DigitalClockFormProps.js"}}
+
+## Views
+
+The `MultiSectionDigitalClock` component can contain three views: `hours`, `minutes`, and `seconds`.
+By default, only the `hours` and `minutes` views are enabled.
+
+You can customize the enabled views using the `views` prop.
+Views will appear in the order they're included in the `views` array.
+
+The `DigitalClock` component is built to work only with `hours` view, providing it with any more views will have no effect as the prop value will be ignored.
+
+{{"demo": "DigitalClockViews.js"}}
+
+## 12h/24h format
+
+The components use the hour format of the locale's time setting, i.e. the 12-hour or 24-hour format.
+
+You can force a specific format using the `ampm` prop.
+
+You can find more information about 12h/24h format in the [Date localization page](/x/react-date-pickers/adapters-locale/#12h-24h-format).
+
+{{"demo": "DigitalClockAmPm.js"}}
+
+## Time steps
+
+By default, the components list the time options in `5` unit (`minutes` or `seconds`) intervals.
+
+You can set the desired interval using the `timeSteps` prop.
+The prop accepts values for `hours`, `minutes`, or `seconds` units.
+
+:::info
+The `DigitalClock` component uses only `minutes` `timeSteps` option, other are ignored.
+:::
+
+{{"demo": "DigitalClockTimeStep.js"}}
+
+## Skip rendering disabled options
+
+With the `skipDisabled` prop, the components don't render options that are not available to the user (e.g. through `minTime`, `maxTime`, `shouldDisabledTime` etc.).
+
+The following example combines these properties to customize which options are rendered.
```

Comment:
```suggestion
The following example combines these properties to customize which options are rendered.
The first component does not show options before 9:00 (the value of `minTime`).
The second one hides options between 11:30 and 13:00 thanks to `shouldDisableTime`.
```

I propose to:
- Add a description of what is hidden. Otherwise, it's not obvious at first look
- Modify the second example, because the use of `%` produce a result similar to what can be obtained with time steps

Reply:
I'm resolving the discussion as it's getting hard to keep track of still open discussions due to the PR events count.
If you still feel room for improvement, feel free to reopen the discussion or start a new one. 👌